src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private Object[] key;
+    private Object[] value;
+    private int sizeTracker = 0;
```

Comment:
```suggestion
    private int sizeTracker;
```
primitives have default values